resources/lang/en/translations.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+return [
+    'translation-navigation-label' => 'Translation Manager',
+    'translation-label' => 'Translation|Translations',
```

Comment:
i just give note for translation faisal, never use " | " or something symbolic like this except punctuation cz translation is reflecting what we wrote from the key. More better if you use " | " outside of translation like in the controller or something else